front/se_project/src/Pages/buyerSignUp.jsx

```diff
@@ -1,78 +1,143 @@
+                          <div className='row justify-content-center' >
+                              <div className='row mb-2'>
+                                  <div class='col row '>
+                                      <label className='col-4 col-form-label' style={{fontSize:'1.1vw'}}>نام و نام‌خانوادگی</label>
+                                      <div className='col-8'> 
+                                          <Input_text type="text" value={names} onChange={(e)=>setNames(e.target.value)}/>
+                                      </div>
+                                  </div>
+                                  <div className='col row'>
+                                      <label className="col-sm-3 col-form-label mx-auto" style={{fontSize:'1.1vw'}}>شماره تماس</label>
+                                      <div className="col-8">
+                                          <Input_text type="text" value={phone} onChange={(e)=>setPhone(e.target.value)}/>
+                                      </div>
+                                  </div>
+                              </div>
+                              <div className='row mb-2'>
+                                  <div class="row col">
+                                      <label class="col-sm-4 col-form-label" style={{fontSize:'1.1vw'}}>رمز عبور</label>
+                                      <div class="col-8">
+                                          <Input_text type="password" value={password} onChange={(e)=>setPassword(e.target.value)}/>
+                                      </div>
+                                  </div>
+                                  <div class="row col">
+                                      <label class="col-sm-3 col-form-label  mx-auto" style={{fontSize:'1.1vw'}}>تکرار رمز عبور</label>
+                                      <div class="col-8">
+                                          <Input_text type="password"/>
+                                      </div>
+                                  </div>
+                              </div>      
+                          </div>
+                      </div>
+                      <div className='container w-75 p-0' >
+                          <div className='justify-content-center mt-0 mx-0' >
+                              <div className='d-flex'>
+                                  <div className='mx-1'>
+                                      <ResetButton/>
+                                  </div>
+                                  <div className='mx-1'>
+                                      <Button width= 'auto' height= 'auto' text= 'ثبت‌نام' fontSize='1.17vw' />
+                                  </div> 
+                              </div>   
+                          </div>
+                      </div>
+                  </form>
+              <Footer/>
+              </div> 
+          </div>  
+     );
 }
  
-export default BuyerSighUp;
+export default BuyerSighUp;
```

Comment:
I think we can delete comments related to class based component.

Reply:
I don not change